.github/workflows/integration.yaml

```diff
@@ -104,3 +104,33 @@ jobs:
           nix shell github:informalsystems/cosmos.nix/gaia-ordered#gaia6-ordered -c cargo \
             test -p ibc-integration-test --features ordered --no-fail-fast -- \
             --nocapture --test-threads=1 test_ordered_channel
+
+  ica-filter-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: cachix/install-nix-action@v15
+        with:
+          install_url: https://nixos-nix-install-tests.cachix.org/serve/vij683ly7sl95nnhb67bdjjfabclr85m/install
+          install_options: '--tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve'
+          extra_nix_config: |
+            experimental-features = nix-command flakes
+      - uses: cachix/cachix-action@v10
+        with:
+          name: cosmos
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - uses: Swatinem/rust-cache@v1
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: -p ibc-integration-test --no-fail-fast --no-run
+      - env:
```

Comment:
We can add `CHAIN_COMMAND_PATH: icad` here to run the test with `icad` instead of `gaiad` without having to hard code the path.